cli/formatter/go-pretty/extra/003_add_no_graphics_style.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/table/style.go b/table/style.go
+index ee87ef8..1babccc 100644
+--- a/table/style.go
++++ b/table/style.go
+@@ -37,6 +37,34 @@ var (
+ 		Title:   TitleOptionsDefault,
+ 	}
+ 
++	// StyleWithoutGraphics defines a style without graphics like below:
++	// col1  col2  col3       col4
++	// 1     477   Elizabeth  12
++	// 3     2804  David      33
++	// 4     1161  William    81
++	// 5     1172  Jack       35
++	// 6     1064  William    25
++	StyleWithoutGraphics = BoxStyle{
++		BottomLeft:       " ",
++		BottomRight:      " ",
++		BottomSeparator:  " ",
++		EmptySeparator:   text.RepeatAndTrim(" ", text.RuneWidthWithoutEscSequences(" ")),
++		Left:             " ",
++		LeftSeparator:    " ",
++		MiddleHorizontal: " ",
++		MiddleSeparator:  " ",
++		MiddleVertical:   " ",
++		PaddingLeft:      " ",
++		PaddingRight:     " ",
++		PageSeparator:    "\n",
++		Right:            " ",
++		RightSeparator:   " ",
++		TopLeft:          " ",
++		TopRight:         " ",
++		TopSeparator:     " ",
++		UnfinishedRow:    "  ",
++	}
```

Comment:
Can we define `StyleWithoutGraphics` in client code without the patch? Why not?

Reply:
Moved to `formatter/style.go`